### PR TITLE
add cards for NFS dashboard

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://issues.redhat.com/browse/RHSTOR-4282
### Status Card
Screenshot from 2023-02-02 13-11-45

### Performance Card
Screenshot from 2023-02-14 18-57-40
Screenshot from 2023-02-14 19-56-51

### Top Clients Card
Screenshot from 2023-02-14 21-49-06
Screenshot from 2023-02-14 21-58-36

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
/hold
metric exporter PR still not merged in upstream...

**Comment by SanjalKatiyar** [comment]
/hold cancel
@bipuladh we can merge it now...

File: packages/ocs/dashboards/network-file-system/status-card/status-card.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+): SubsystemHealth => {
+  if (loading) {
+    return { state: HealthState.LOADING };
+  }
+  if (loadError) {
+    return { state: HealthState.NOT_AVAILABLE };
+  }
+  if (!!state && state !== '0') {
+    return { state: HealthState.OK };
+  }
```
Comment (bipuladh) [inline packages/ocs/dashboards/network-file-system/status-card/status-card.tsx:28]: If total worker threads is not `0` then there is a Health issue?

**Review by bipuladh — COMMENTED** [review]
lgtm

**Pull request merged**